unix-cheat-sheet.tex

```diff
@@ -150,9 +150,9 @@ \subsection*{Input output redirection}
                                             & output of \cv{cmd1} to the input of \cv{cmd2}\\
 \shcmd{\cv{cmd}}{> file}                    & runs \cv{cmd} and redirects output to \cv{file},\\
                                             & content of \cv{file} is overwritten\\
-\shcmd{\cv{cmd}}{>\null> file}              & runs \cv{cmd} and redirects output to \cv{file},\\
-                                            & the output is appended to \cv{file}\\
-\shcmd{\cv{cmd}}{< file}                    & runs \cv{cmd} and redirects \cv{file} to its input
+\shcmd{\cv{cmd}}{>\null> file}              & like \shcmd{}{>>} but appended output to \cv{file}\\
+\shcmd{\cv{cmd}}{< file}                    & runs \cv{cmd} and redirects \cv{file} to its input\\
+\shcmd{\cv{cmd}}{<<< text}                  & runs \cv{cmd} with input \cv{text}
```

Comment:
This actually is a here string. Maybe we should add here documents (`<<delim`) too.

Reply:
True. But I misspoke. But I think heredoc needs too much explanation due to the delimited to be useful in such a short form.